src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+0.1.4 (unreleased)
+++++++++++++++++++++
+* hostname command bug fixes (#2946, #2947, #2949)
```

Comment:
I wouldn't expect anyone to go cross-reference closed GitHub issues to figure out what this means. You should summarize the changes from a user perspective--especially the breaking change.